### ISSUE TITLE
LinePlusBarChart doesn't hide scale when deactivating right axis series.

### ISSUE THREAD

**Issue by tapia** [novus/nvd3#917]
In the [Line Plus Bar Chart example](http://nvd3.org/examples/linePlusBar.html), if you click on the left axis legend to hide that series, the scale is correctly hidden, but if you do the same in the right axis legend, the scale doesn't hide and shows the default [-1, 1] range. It should hide, like the left side one.

**Comment by liquidpele** [comment]
that example is using a very old version if you try the latest development then I believe that bug has been fixed

**Comment by tapia** [comment]
I've just tried with 1.7.1 and the bug is still there.

**Comment by liquidpele** [comment]
I just tried it with the development branch and I couldn't reproduce... can you provide an example on jsfiddle using the latest development code that shows it happening?

**Comment by tapia** [comment]
I didn't try the development branch, I tried with the 1.7.1 version, which is the stable version (at least, the nvd3 site says so).

**Comment by liquidpele** [comment]
Yep, but the development branch has a lot more changes, it's about to go out as 1.8.x soon probably.  

If you have the problem on the dev code, then please create a jsfiddle so I can see the exact scenario since I can't seem to reproduce myself.  Thanks!

**Comment by liquidpele** [comment]
ping!   any update on this?
